/**:
    ros__parameters:
        mqtt:
            client:
                protocol: 4      # MQTTv311
            connection:
                host: localhost # these are set in the launch file, so we could have args from outside
                port: 1889
                keepalive: 30
            # tls:
            #     tls_insecure:
            #     ca_certs:
            #     certfile:
            #     keyfile:
            #     cert_reqs:
            #     tls_version:
            #     ciphers:
            # account:
            #     username:
            #     password:
            # message:
            #     max_inflight_messages:
            #     max_queued_messages:
            #     message_rery:
            # will:
            #     topic:
            #     payload:
            #     qos:
            #     retain:
            #     properties:
            namespace: "waraps/unit/surface/real/evolo"
            # ros namespacing should be done in the launch file
            # each string here will be a topic on both sides, under their respective namespaces
            to_mqtt:
              [
                #lv1
                "/local/map",
                 #lv1
                
                

              ]
            to_ros:
              [
               #lv2
                "exec/command",
              ]